modules/instrument_builder/jsx/react.app.js

```diff
@@ -0,0 +1,485 @@
+TabPane = React.createClass({
+    render: function() {
+        var classList = "tab-pane";
+        if(this.props.Active) {
+            classList += " active"
+        }
+        return (
+            <div className={classList} id={this.props.TabId}>
+                <h1>{this.props.Title}</h1>
+                <br />
+                {this.props.children}
```

Comment:
Can this be wrapped in a div with appropriate styling instead of having a br tag?